src/Kdyby/ElasticSearch/Diagnostics/Panel.php

```diff
@@ -104,6 +104,11 @@ public function getPanel()
 				return array();
 			}
 
+			if ($object instanceof Elastica\Request) {
+				$j = new Elastica\JSONFormat('  ', "\n"); // indent and linebreak characters
```

Comment:
Now I am ready to upgrade to a request JSON syntax highlighting. Class Elastica \ JSONFormat no longer needed.